src/pages/dar_collection_review/ApplicationInformation.js

```diff
@@ -53,10 +61,37 @@ const generateLabelSpanContents = (labelValue, key,  spanValue, isLoading) => {
   );
 };
 
+// function to generate application details content
+const dynamicRowGeneration = (rowElementMaxCount, appDetailLabels, loading) => {
+  // lodash/fp filter (non-empty string, non-empty object, non-empty array, booleans)
+  const labels = filter(label => {
+    return (typeof label.value === 'boolean' || !isEmpty(label.value));
+  }) (appDetailLabels);
+
+  const labelArray = labels.map(label => {
+    if (typeof label.value === 'boolean') {
+      // Inject "Yes" / "No" for booleans
+      return generateLabelSpanContents(label.title, label.key, (label.value ? 'Yes' : 'No'), loading);
```

Comment:
Just double checking - we want to display all booleans regardless of whether they were true or false, correct?

Reply:
That seems right to me. A user can select true/false in multiple DAR questions, so that should be reflected.

Reply:
For now yes. We haven't figured out whether or not we should only show the true values or show all, so to be safe just stick with logic you have here. We can always update the logic if need be with a new ticket.